.vscode/settings.json

```diff
@@ -9,5 +9,8 @@
   },
   "editor.formatOnSave": true,
   "editor.formatOnPaste": true,
-  "editor.defaultFormatter": "esbenp.prettier-vscode"
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "[python]": {
```

Comment:
hrm, do we need to check this in? feels kind of weird since these vscode settings are used by the rest of the team, but python is only used for some light scripting for the mtg thing